docs/Change_Log.md

```diff
@@ -1,5 +1,26 @@
 # Change Log
 
+## Updated Command Permissions
+
+#### Apr 27, 2022
+
+Application command permissions have been updated to add more granular control and access to commands in Discord. You can read the major changes below, and [the updated documentation](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/permissions) for details.
+
+#### Breaking changes
+
+- Bearer tokens are now required to edit command permissions. Bearer tokens are tokens tied to an authenticating user's permissions, and can be [retrieved using OAuth](#DOCS_TOPICS_OAUTH2). The user must have permission to manage the guild and roles.
+- [`applications.commands.permissions.update`](#DOCS_TOPICS_OAUTH2/shared-resources-oauth2-scopes) scope was added as a requirement to edit command permissions.
+- Disabled the batch editing endpoint ([`PUT /applications/{application.id}/guilds/{guild.id}/commands/permissions`](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/batch-edit-application-command-permissions)).
```

Comment:
[![](https://t.pics/eM3jfshPukI.png)](https://canary.discord.com/channels/613425648685547541/965807097356910642/968562854724796467)
Thank you so much snek for fixing the v2 permissions bug with batch editing yesterday... oh wait now it's completely gone without warning the next day... Why even fix it if it's just going away the next day without warning?!